### ISSUE TITLE
with_deleted when both relations deleted does not work

### ISSUE THREAD

**Issue by faouzzz** [ActsAsParanoid/acts_as_paranoid#77]
```ruby
@offices = Office.with_deleted.includes({client_with_deleted: [:offices]}, :slots)
                   .order('clients.name, offices.name')

@offices.each do |office|
  office_count = office.client_with_deleted.offices.count
end
```

raise **undefined method `offices' for nil:NilClass** when office and client are deleted.

**Comment by mvz** [comment]
This looks similar to #76 so will hopefully be fixed in the next release as well.

**Comment by mvz** [comment]
This was fixed in #147.
